src/fragments/lib/auth/flutter/credential_storage/10_credential_storage.mdx

```diff
@@ -0,0 +1,37 @@
+<BlockSwitcher>
+
+<Block name="Stable (Mobile)">
+
+Amplify Flutter uses the underlying storage mechanisms used by amplify-android and amplify-ios on their respective platforms, and does not offer customization.
+
+</Block>
+
+<Block name="Developer Preview (Mobile, Web & Desktop)">
+
+Amplify Flutter's developer preview for web and desktop support offers a `SecureStorageInterface` interface which can be used to customize how authentication related information is stored on each supported platform. An instance of this interface can be passed into the `AmplifyAuthCognito` constructor.
+
+An implementation of the `SecureStorageInterface` can be used directly if you'd like to control underlying methods such as `read` and `write`. If you want to define configuration values only, you can use the `AmplifySecureStorage` class.
+
+For example, the `AmplifyAuthCognito` plugin uses IndexedDB to persist tokens and credentials, so that this data remains available when the user refreshes or closes their browser. If you'd like to only persist these values in memory, you can change the persistence option for web:
```

Comment:
```suggestion
For example, the `AmplifyAuthCognito` plugin uses IndexedDB to persist tokens and credentials on web so that this data remains available when the user refreshes or closes their browser. If you'd like to only persist these values in memory, you can change the persistence option like so:
```